Miguel-Leite/challenge-wedev
# Adds custom Docker configuration

# Summary:
This feature adds a custom Docker configuration to the application.

# Problem:
Previously, Docker configuration for the application was limited to a default set of settings. This did not allow for specific customizations for different environments or project needs.

# Solution:
This feature adds a custom setting that allows users to customize the settings according to their needs.

# Changes:
- Adds new Docker configuration files
- Updates the Docker build script to include the new settings

# Impact:
There is no impact on dependencies or other parts of the project.

# Review Steps:
1. Make sure the documentation is up to date with information about the new settings.
2. Verify that your custom Docker configuration works correctly.

# Comments:
- No automated tests have been performed at this time.
- The README has not been updated yet, but will be updated before merging.
